test/graph_test.cpp

```diff
@@ -38,15 +39,26 @@ TEST(GraphTestSuite, IFconnectedComponentsAlgRunTHENupdateLocked) {
     in >> a >> b;
     g.update({{a, b}, INSERT});
   }
-  g.set_cumul_in(curr_dir + "/res/multiples_graph_1024.txt");
+  g.set_verifier(std::make_unique<FileGraphVerifier>(curr_dir + "/res/multiples_graph_1024.txt"));
   g.connected_components();
   ASSERT_THROW(g.update({{1,2}, INSERT}), UpdateLockedException);
   ASSERT_THROW(g.update({{1,2}, DELETE}), UpdateLockedException);
 }
 
 TEST(GraphTestSuite, TestRandomGraphGeneration) {
   generate_stream();
-  GraphVerifier graphVerifier {};
+  ifstream in { "./cumul_sample.txt" };
```

Comment:
What exactly is this test accomplishing? Just asserting that creating a graph verifier doesn't crash?